src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml.cs

```diff
@@ -65,6 +65,8 @@ public AdvancedOptionPageControl(OptionStore optionStore, IComponentModel compon
 
             BindToOption(Split_string_literals_on_enter, SplitStringLiteralOptions.Enabled, LanguageNames.CSharp);
 
+            BindToOption(Move_to_end_of_expression_on_semi_colon, FeatureOnOffOptions.MoveToEndOfExpressionOnSemiColon);
```

Comment:
Should this be marked as C#-specific?

(I don't think it actually matters, but I noticed it was the odd one out among most of the other options in this file.)

Reply:
Specifying language name is for per language settings, but this one isn't since it doesn't apply to VB I didn't go that way.

I am 50% confident that's the logical thing to do, but there is a 50% chance that the preference is for this to be a per language option precisely because it doesn't apply to VB.

🍿